docs/src/docs/asciidoc/guides/how-to-custom-claims-authorities.adoc

```diff
@@ -0,0 +1,49 @@
+[[how-to-extension-grant-type]]
+= How-to: Add authorities to custom claims in JWT-based access tokens
```

Comment:
Please update to -> "How-to: Add authorities as custom claims in JWT access tokens"